spec/controllers/objects_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe ObjectsController, type: :controller do
```

Comment:
Controller specs are deprecated in rails. Can you make request specs instead?

Reply:
converted to request specs and added context blocks around tests